### PR TITLE
Only active users for rsvp list + RSVP CSV

### PR DESCRIPTION
**Pull request by iambibhas**

### CODE REVIEW

File: funnel/views/project.py [hunk 1]
```diff
@@ -460,6 +459,55 @@ def rsvp_transition(self):
...
+
+        outfile.seek(0)
+        return Response(
+            outfile.getvalue(),
+            content_type='text/csv',
+            headers=[
+                (
+                    'Content-Disposition',
+                    'attachment;filename="{filename}.csv"'.format(
+                        filename='rsvp-{project}-{state}'.format(
```
Comment (jace) [inline funnel/views/project.py:485]: Filename prefix should be `participants-`

File: funnel/views/project.py [hunk 2]
```diff
@@ -460,6 +459,55 @@ def rsvp_transition(self):
...
+                            project=make_name(self.obj.title), state=state
```
Comment (jace) [inline funnel/views/project.py:486]: `project=self.obj.name`

Comment (iambibhas) [reply funnel/views/project.py:486]: can't do this because project names are mostly years. nor verbose.

Comment (jace) [reply funnel/views/project.py:486]: `profile-project`?

Comment (iambibhas) [reply funnel/views/project.py:486]: That's what make_name on title does, right? Our do you want to avoid the function call?

**Pull request merged**